packages/core/lib/json-payload.js

```diff
@@ -1,7 +1,20 @@
 const jsonStringify = require('@bugsnag/safe-json-stringify')
+const filterPaths = [
+  // report
+  'events.[].app',
+  'events.[].metaData',
+  'events.[].user',
+  'events.[].breadcrumbs',
+  'events.[].request',
+  'events.[].device',
+  // session
+  'device',
+  'app',
+  'user'
+]
```

Comment:
should these just be two separate arrays? We have the two separate methods that use this array and it seems like there is no sharing between the two sets?